### PR TITLE
Fix code blocks in quickstart list

### PR DESCRIPTION
**Pull request by remram44**
See problem: https://streamz.readthedocs.io/en/latest/#quickstart

screenshot of current readthedocs

Fixed:

screenshot of fixed version

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -84,14 +84,18 @@ environment but rather for experimentation, learning, or new feature development
 Its most common use would be to interact with the streamz example jupyter notebooks. Lets walk through the steps needed for this.
 
 - Build the Docker container
-.. code-block:: bash
-$ docker/build.sh
+
+  .. code-block:: bash
```
Comment (martindurant) [inline docs/source/index.rst:88]: I don't think this directive (and below) should be indented

**Review by martindurant — COMMENTED** [review]
Thanks; not sure how this got messed up.
Minor comment.

Have you built the HTML locally?

File: docs/source/index.rst (hunk 1 above)
Comment (remram44) [reply docs/source/index.rst:88]: It does, that was the problem :) That is how you indicate that the code block is part of the list item.

**Comment by remram44** [comment]
I did build locally, that is where the second screenshot comes from (the first one is from the current readthedocs page).

**Comment by martindurant** [comment]
OK, sure that convinces me :)

Thanks!

**Pull request merged**